tests/test_popularity.py

```diff
@@ -0,0 +1,195 @@
+# -*- coding: utf-8 -*-
+
+from mabwiser.mab import LearningPolicy, NeighborhoodPolicy
+from tests.test_base import BaseTest
+
+
+class PopularityTest(BaseTest):
```

Comment:
Since Popularity inherits from EpsilonGreedy it also inherits the epsilon property. Should there be a unit test to verify that manually changing epsilon has no impact on the predictions?

Reply:
Good point! I will add

Reply:
Btw, notice that standard bandits mab = MAB(LP.Popularity()) would allow you to access epsilon manually :) We close that door long time ago.
I can still create a hacky test .

Reply:
I think there are two issues - LP.Popularity does not have an epsilon value and there should be a test in test_invalid to verify that if you try to pass an epsilon value to LP.Popularity it raises an error, but mabwiser.popularity.Popularity does have epsilon that could be accessed as mab._imp.epsilon so we should verify that nothing happens if the person accesses the implementor to change that value

Reply:
done, and done for both tests.